lib/Doctrine/Adapter/Statement.php

```diff
@@ -20,112 +20,85 @@
  */
 
 /**
- * Doctrine_Adapter_Statement
+ * Doctrine_Adapter_Statement.
  *
  * @author      Konsta Vesterinen <kvesteri@cc.hut.fi>
  * @license     http://www.opensource.org/licenses/lgpl-license.php LGPL
- * @package     Doctrine
- * @subpackage  Adapter
- * @link        www.doctrine-project.org
+ *
+ * @see        www.doctrine-project.org
  * @since       1.0
+ *
  * @version     $Revision: 7490 $
  */
 abstract class Doctrine_Adapter_Statement
 {
     /**
-     * bindValue
+     * bindValue.
      *
-     * @param string $no 
-     * @param string $value 
-     * @return void
+     * @param string $no
+     * @param string $value
      */
     public function bindValue($no, $value)
-    { }
+    {
+    }
 
     /**
-     * fetch
+     * fetch.
      *
      * @see Doctrine_Core::FETCH_* constants
-     * @param integer $fetchStyle           Controls how the next row will be returned to the caller.
```

Comment:
this is weird: are those parameters not used or sub-implementations are defining them?

Reply:
ping @connorhu @alquerci

Reply:
@thePanz see #114